From 70aaad841a5067db2737ea658532eb92d9376888 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Fri, 24 Jul 2015 17:19:06 +0200
Subject: [PATCH 2/2] Update information about DEB_*_ARCH_* variables

Mention DEB_*_ARCH_BITS and DEB_*_ARCH_ENDIAN. Introduced in dpkg 1.15.4.
---
 policy.sgml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/policy.sgml b/policy.sgml
index 72a2505..b7f0464 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2197,6 +2197,12 @@ zope.
 		<tt>DEB_*_ARCH_CPU</tt> (the Debian CPU name)
 	    </item>
 	    <item>
+		<tt>DEB_*_ARCH_BITS</tt> (the Debian CPU pointer size in bits)
+	    </item>
+	    <item>
+		<tt>DEB_*_ARCH_ENDIAN</tt> (the Debian CPU endianness)
+	    </item>
+	    <item>
 		<tt>DEB_*_ARCH_OS</tt> (the Debian System name)
 	    </item>
 	    <item>
-- 
2.5.0.rc2.392.g76e840b

